### DISCUSSION TITLE
Welcome to vaahcms-module-store Discussions!

### DISCUSSION THREAD

**Discussion by themodernpk** [webreinvent/vaahcms-module-store#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SuryaJD** [comment]
I would like to contribute. How can join you as a open source contibuter.

**Reply by themodernpk** [reply]
@SuryaJD join us on Slack using the following:
https://join.slack.com/t/vaah/shared_invite/zt-wgvx75rr-tuAhGtjRweCR~DEVSTFcSQ
